raiden/smart_contracts/ChannelManagerContract.sol

```diff
@@ -9,6 +9,17 @@ import "./ChannelManagerLibrary.sol";
 contract ChannelManagerContract {
     using ChannelManagerLibrary for ChannelManagerLibrary.Data;
     ChannelManagerLibrary.Data data;
+    /// @dev All open channels for a specific token
```

Comment:
There is no need for a tag on top of variables, please remove it since it's not parsed by the compiler at all and would simply appear as is, which is not what we want.